lms/djangoapps/teams/static/teams/js/views/edit_team.js

```diff
@@ -0,0 +1,192 @@
+                   this.set(this.teamDescriptionField, '.team-required-fields');
+                   this.set(this.optionalDescriptionField, '.team-optional-fields');
+                   this.set(this.teamLanguageField, '.team-optional-fields');
+                   this.set(this.teamCountryField, '.team-optional-fields');
+                   return this;
+               },
+
+               set: function(view, selector) {
+                   var viewEl = view.$el;
+                   if (this.$(selector).has(viewEl).length) {
+                       view.render().setElement(viewEl);
+                   } else {
+                       this.$(selector).append(view.render().$el);
+                   }
+               },
+
+               createTeam: function () {
+                   var teamName = this.teamNameField.fieldValue();
+                   var teamDescription = this.teamDescriptionField.fieldValue();
+                   var teamLanguage = this.teamLanguageField.fieldValue();
+                   var teamCountry = this.teamCountryField.fieldValue();
+
+                   var data = {
+                       course_id: this.courseId,
+                       topic_id: this.topicId,
+                       name: teamName,
+                       description: teamDescription,
+                       language: _.isNull(teamLanguage) ? '' : teamLanguage,
+                       country: _.isNull(teamCountry) ? '' : teamCountry
+                   };
+
+                   var validationResult = this.validateTeamData(data);
+                   if (validationResult.status === false) {
+                       this.showMessage(validationResult.message, validationResult.srMessage);
+                       return;
+                   }
+
+                   var view = this;
+                   var options = {
+                       wait: true,
+                       success: function () {
+                           view.goBackToTopic();
+                       },
+                       error: function () {
+                           var message = gettext('An error occurred. Please try again.');
+                           view.showMessage(message, message);
+                       }
+                   };
+                   this.teamModel.save(data, options);
+               },
+
+               validateTeamData: function (data) {
+                   var status = true,
+                       message = gettext("Check the highlighted fields below and try again.");
+                   var srMessages = [];
+
+                   this.teamNameField.unhighlightField();
+                   this.teamDescriptionField.unhighlightField();
+
+                   if (_.isEmpty(data.name.trim()) ) {
+                       status = false;
+                       this.teamNameField.highlightFieldOnError();
+                       srMessages.push(
+                           gettext("Please enter team name")
```

Comment:
Can we follow our UI style and also remove the "Please" from these screen reader messages? I think punctuation is helpful too, although @clrux can confirm.
"Enter the team name."